test/babelish/test_android2csv.rb

```diff
@@ -28,6 +28,26 @@ def test_initialize
       assert_equal filenames, converter.filenames
     end
 
+  def test_special_chars
+    csv_filename = "./android_special_chars_output.csv"
+    filenames = "test/data/android_special_chars.xml"
+    headers = %w{"constants german"}
+
+    converter = Babelish::Android2CSV.new({
+      :csv_filename => csv_filename,
+      :headers => %w{variables german},
+      :filenames => [filenames]
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.